README.md

```diff
@@ -47,6 +47,24 @@ To create a package from scratch, refer to the nixpkgs documentation available a
 
 ## Bundles and jobs
 
+Bundles are defined in your `config.nix` file. If you didn't change this part of the auto-generated file, you have a single bundle called "default".
+Bundles are used to create sets of compatible packages. You can override the version of some packages and you can explicitly exclude some incompatible packages.
+
+Jobs represent buildable outputs. You can build any package in `coqPackages` (including any package defined in your `.nix/coq-overlays` directory) with the following command:
+
+```
+nix-build --argstr job PACKAGENAME
```

Comment:
One can replace `PACKAGENAME` with:
- `_allJobs` to compile all coq packages that are explicitly mentioned in the `config.nix` and not explicitly excluded
- `_all` to compile all coq packages that are not explicitly excluded